src/patternfly/components/LabelGroup/examples/LabelGroup.md

```diff
@@ -0,0 +1,402 @@
+---
+id: 'Label group'
+beta: true
+section: components
+cssPrefix: pf-c-label-group
+---
+
+## Examples
+### Basic
+```hbs
+{{#> label-group label-group--id="label-group-basic"}}
+  {{#> label-group-list label-group-list--attribute='aria-label="Label group list"'}}
```

Comment:
Group of labels

Reply:
as a suggestion for the aria-label

Reply:
Sure, though for what it's worth, this matches the chip group. Do you think they need to be different?

Reply:
Yeah i think it would benefit the user because IMO its easier to comprehend when the screen reader is reading it